docs/source/contributing/releasing.adoc

```diff
@@ -99,3 +99,18 @@ For example:
 $ cd scripts/release
 $ ./issue-list.sh -r minishift -m 9
 ----
+
+To update the {project} cask we need to do the following tasks:
```

Comment:
Add another section saying, `Updating Homebrew-Cask`

Reply:
Update `To update the {project} cask we need to do the following tasks:` to `To update the {project} cask manually, we need to do the following tasks:`